runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -331,18 +379,18 @@ class LayoutFFITypeHelpers
 	/* @brief Encode a signature string to a struct representing the signature to be used
 	 * to handle the argument or return value in the upcall.
 	 *
-	 * @param cSignature[in] A pointer to a preprocessed signature string
+	 * @param cSignature[in/out] A pointer to a preprocessed signature string, which points
+	 * to the end of the signature string upon return.
```

Comment:
This is not an "out" parameter, so there's nothing to describe for callers.

Reply:
Removed.